src/capacitor.ts

```diff
@@ -0,0 +1,135 @@
+const platforms: Fig.Suggestion[] = [
+  { name: "android", icon: "fig://icon?type=android" },
+  { name: "ios", icon: "fig://icon?type=apple" },
+];
+
+const helpOption: Fig.Option = {
+  name: ["--help", "-h"],
+  description:
+    "Output usage information. Can be used with individual commands too",
+};
```

Comment:
If you add this property the option become available globally to all subcommands so it no longer needs to be repeated and can be moved directly in the completion object
```suggestion
  name: ["--help", "-h"],
  description:
    "Output usage information. Can be used with individual commands too",
  isPersistent: true,
};
```

Reply:
Thanks that is way better 👍